packages/gatsby/src/utils/get-latest-gatsby-files.ts

```diff
@@ -23,29 +24,52 @@ export interface IAPIResponse {
   ssr: Record<string, any>
 }
 
+const _fetchFile = async (root: string, fileName: string): Promise<any> => {
+  try {
+    const { data } = await axios.get(`${root}${fileName}`, {
+      timeout: 5000,
+    })
+    return data
+  } catch (e) {
+    return null
+  }
+}
+
 const _getFile = async <T>({
   fileName,
   outputFileName,
   defaultReturn,
+  tryGithubBeforeUnpkg,
+  forcedContent,
 }: {
   fileName: string
   outputFileName: string
   defaultReturn: T
+  tryGithubBeforeUnpkg?: boolean
+  forcedContent?: string
 }): Promise<T> => {
   let fileToUse = path.join(ROOT, fileName)
-  try {
-    const { data } = await axios.get(`${UNPKG_ROOT}${fileName}`, {
-      timeout: 5000,
-    })
 
+  let dataToUse = forcedContent
+
+  if (!dataToUse && tryGithubBeforeUnpkg) {
+    dataToUse = await _fetchFile(GITHUB_ROOT, fileName)
+  }
```

Comment:
optionally this uses github as source of content, to make it faster to deliver adapter version manifest changes to users - using unpkg require npm publish which currently is not the fastest process